tests/test_errors.py

```diff
@@ -170,11 +170,88 @@ def get_model(compile, meta, compile_kwargs):
 
     est = KerasRegressor(
         model=get_model,
-        loss="categorical_crossentropy",
+        loss="sparse_categorical_crossentropy",
```

Comment:
Why this change? This seems rather significant.

Reply:
This shouldn't have been changed here.

I did change it in several other places because previously, SciKeras would introspect your models actual loss function to one-hot-encode the target if `iscategoricalcrossentropy(loss)`.  Since we now require specifying the loss like `KerasClassifier(loss="categorical_crossentropy")` for that to work, we are faced with a choice:
1. Edit all tests to specify the loss as part of the constructor.
2. Edit the hardcoded loss function.

I opted for the latter.